sections/designRules.md

```diff
@@ -745,9 +766,10 @@ Note: security controls for signing and encrypting of application level messages
       </dd>
       <dt>Rationale</dt>
       <dd>
-         <p>Even when using TLS connections, information in URIs is not secured. URIs can be cached and logged outside of the servers controlled by clients and servers. Any information contained in them should therefore be considered readable by anyone with access to the network (in the case of the internet, the whole world) and MUST NOT contain any sensitive information. This includes client secrets used for authentication, privacy sensitive information such as BSNs or any other information which should not be shared.
+         <p>Even when using TLS connections, information in URIs is not secured. URIs can be cached and logged outside of the servers controlled by clients and servers. Any information contained in them should therefore be considered readable by anyone with access to the network (in the case of the internet, the whole world) and MUST NOT contain any sensitive information. This includes client secrets used for authentication, privacy sensitive information or any other information which should not be shared.
          <p>Be aware that queries (anything after the '?' in a URI) are also part of a URI.
       </dd>
+      <p class="note">It is up to you what the definition of sensitive means for your context. The term sensitive is deliberatly not defined in this document.</p>
```

Comment:
Spec tekst heeft geen target audience, dus "you" moeten we vermijden (ik kom er achter dat we wat andere gevallen hiervan hebben, dus die zal ik ook opruimen).

```suggestion
      <p class="note">The term sensitive is deliberately not defined in this document as its definition depends on its context.
```